bounty/bounty_agent.py

```diff
@@ -126,26 +124,26 @@ def get_bounty(self):
 
         self.logger.info(LONG_DOUBLE_LINE)
 
-        if receipt['status'] == 1:
+        if res_tx.receipt['status'] == 1:
```

Comment:
this check is reduntant with current `transaction_method` implementation

Reply:
It was left for future changes in transaction_method in order to not forget about it) 
Anyway - removed